cli/msg.fbs

```diff
@@ -279,6 +281,12 @@ table EnvironRes {
   map: [KeyValue];
 }
 
+table Deps {}
+
+table DepsRes {
+  data: [ubyte];
```

Comment:
wasn't sure if this should be type `string` or not, since it could be arbitrarily large